config.reek

```diff
@@ -20,6 +20,13 @@ LongParameterList:
   exclude:
   - initialize
 
+UncommunicativeMethodName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
+UncommunicativeVariableName:
+  accept:
+    - s3 # elb, rds, ec2, sqs are all fine
```

Comment:
Did you try `ec2`?  Based on the documentation in Reek, that would be flagged as an invalid variable/method name.